config/install/migrate_plus.migration.joomla_node.yml

```diff
@@ -19,6 +19,17 @@ process:
   sticky:
     plugin: default_value
     default_value: 0
+  langcode:
```

Comment:
I don't think we can migrate translations until Internationalization is enabled and configured.